crypto-ffi/src/wasm/mod.rs

```diff
@@ -1082,11 +1082,16 @@ impl From<core_crypto::e2e_identity::E2eiDumpedPkiEnv> for E2eiDumpedPkiEnv {
 
 #[wasm_bindgen]
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+/// Configuration object for new conversations
 /// see [core_crypto::prelude::MlsConversationConfiguration]
 pub struct ConversationConfiguration {
-    ciphersuite: Option<Ciphersuite>,
+    #[wasm_bindgen(readonly)]
+    /// Conversation ciphersuite
+    pub ciphersuite: Option<Ciphersuite>,
     external_senders: Vec<Vec<u8>>,
```

Comment:
`external_senders` is not readonly?

Reply:
I do see the other `externalSenders` getter implementation, but I would expect that right now the JS version of a conversation configuration has a read-only `externalSenders` getter and a writeable `external_senders` field.

Reply:
wasm-bindgen only generates getters and setters for `pub` items. Before this PR, we had an interface that was used to provide docs, and not much more than that.

Reply:
Note that there is a getter in the `impl` block, so we still have a getter, but not a setter -- essentialy what `readonly` would give you.

Reply:
Ah, ok. You're right, I missed that it wasn't `pub`.